### PR TITLE
Resolver for gallery of the week winners

### PR DESCRIPTION
**Pull request by Robinnnnn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,4 +27,6 @@ _internal*
 *.mp4
 *.jpg
 
-refresh.json
\ No newline at end of file
+refresh.json
+
+bin/*
```
Comment (Robinnnnn) [inline .gitignore:32]: ignore binaries generated locally

File: graphql/resolver/schema.resolvers.go [hunk 2]
```diff
@@ -515,6 +515,10 @@ func (r *queryResolver) GeneralAllowlist(ctx context.Context) ([]*persist.ChainA
 	return resolveGeneralAllowlist(ctx)
 }
 
+func (r *queryResolver) GalleryOfTheWeekWinners(ctx context.Context) ([]*model.GalleryUser, error) {
+	return publicapi.For(ctx).Misc.GetGalleryOfTheWeekWinners(ctx)
```
Comment (radazen) [inline graphql/resolver/schema.resolvers.go:519]: You'll want to convert your `[]sqlc.User` to a `[]*model.GalleryUser` here. The `userToModel` function will handle the conversion!

File: graphql/schema/schema.graphql [hunk 3]
```diff
@@ -376,6 +376,7 @@ type Query {
     collectionTokenById(tokenId: DBID!, collectionId: DBID!): CollectionTokenByIdOrError
     communityByAddress(communityAddress: ChainAddressInput!, forceRefresh: Boolean): CommunityByAddressOrError
     generalAllowlist: [ChainAddress!]
+    galleryOfTheWeekWinners: [GalleryUser!] @goField(forceResolver: true)
```
Comment (radazen) [inline graphql/schema/schema.graphql:379]: I think you can drop the `@goField(forceResolver: true)` since queries will generate resolvers by default.

**Review by radazen — COMMENTED** [review]
Left minor comments! Happy to re-review ASAP.

**Pull request merged**